ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/signing/fileBased/FileBasedSignerHelper.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core.signing.fileBased;
+
+import tech.pegasys.ethsigner.core.signing.CredentialTransactionSigner;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import com.google.common.base.Charsets;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.web3j.crypto.CipherException;
+import org.web3j.crypto.Credentials;
+import org.web3j.crypto.WalletUtils;
+
+public class FileBasedSignerHelper {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  public static CredentialTransactionSigner getSigner(final FileBasedSignerConfig config) {
+    String password;
+    try {
+      password = readPasswordFromFile(config);
+    } catch (IOException e) {
+      LOG.error(
+          "Error when reading the password from file using the following config:\n"
+              + config.toString()
```

Comment:
Be good to include the file password file path in the log. Also be nice to use the {} place holders for the strings and just pass through the exception as is last arg for these logs messages.

Reply:
Done!